ISSUE_TEMPLATE

```diff
@@ -0,0 +1,41 @@
+Before filing an issue:
+-----------------------
+- Attempt to reproduce the issue using 
+  [this sample application](Examples/SnapinsSDKExample/)
+- Search [existing issues](https://github.com/forcedotcom/ServiceSDK-iOS/issues), 
+  including issues that are closed.
+- Review the "Known Issues" 
+  [section of our release documentation](https://github.com/forcedotcom/ServiceSDK-iOS/releases) 
+  for your issue.
+- If this is an implementation-specific question please refer to the
+  [Snap-Ins for iOS Developer Guide](https://developer.salesforce.com/docs/atlas.en-us.noversion.service_sdk_ios.meta/service_sdk_ios/servicesdk_ios_dev_guide.htm)
+  or the [Service Cloud Snap-Ins API Reference](http://forcedotcom.github.io/ServiceSDK-iOS/).
+- This issue tracker is intended for bugs and iOS Snap-Ins SDK
```

Comment:
This seems like less of a checklist item and more like a general note. (Also making it a lowercase "i" because we only capitalize the "I" when using Title Case... Not super intuitive, I know...)
```suggestion

**NOTE**: This issue tracker is intended for bugs and iOS Snap-ins SDK
```